packages/block-library/src/group/variations.js

```diff
@@ -4,12 +4,21 @@
 import { __ } from '@wordpress/i18n';
 
 const variations = [
+	{
+		name: 'group',
+		title: __( 'Group' ),
+		description: __( 'Blocks shown in a column.' ),
+		attributes: { layout: { type: 'default' } },
+		scope: [ 'inserter', 'transform' ],
```

Comment:
```suggestion
		scope: [ 'transform' ],
```

To avoid duplicated Group block in the inserter:
![CleanShot 2022-01-04 at 18 42 12](https://user-images.githubusercontent.com/240569/148076408-f59990b1-42c6-4b8e-88f0-ce77c7cd69f8.png)

Reply:
We could also set this variation as a default one with `isDefault` set to `true` so it doesn't show up twice in the inserter. However, it might have some other consequences because this variation overrides the title and description.